app/pages/patientnew/patientnew.js

```diff
@@ -331,14 +334,14 @@ var PatientNew = React.createClass({
     if (!formValues.birthday) {
       validationErrors.birthday = IS_REQUIRED;
     }
-    else if (!datetimeUtils.isValidDate(formValues.birthday)) {
+    else if (!sundial.isValidDate(formValues.birthday)) {
```

Comment:
Same validation issue. I think this is the deeper bug that Kent was looking at - moment will reject dates like 2014-02-40 (no month has 40 days), but it will not reject 2014-02-30 (even though Feb. never has 30 days), and it might be because it is trying to be "flexible" about whether months are 0-indexed or not, I don't know. I do think we need to figure out a way to invalidate cases like this. It may mean adding a more robust validation function (which we may not want to use everywhere, but only for form validation, because it will probably be more expensive).